### ISSUE TITLE
AD users not able to login even with valid credentials.

### ISSUE THREAD

**Issue by Pkumar22in** [grokability/snipe-it#12496]
### Debug mode

- [X] I have enabled debug mode
- [X] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

### Describe the bug

Hi,

In my installation I have Ldap Integrated which was working fine for all my users for years. Now suddenly my admin team(5 people) are not able to login. They have the right credentials, I tried testing the ldap synch and it went fine as shown below.

<img width="850" alt="Screenshot 2023-02-09 at 5 19 12 PM" src="https://user-images.githubusercontent.com/94165474/217805599-ff428663-7b65-4008-b9ce-2e117ae861a2.png">

But when I test with a user who is very much in same DN Below error I get on ldap page.
<img width="840" alt="Screenshot 2023-02-09 at 5 19 46 PM" src="https://user-images.githubusercontent.com/94165474/217805799-ae5cef03-099d-4794-8749-72e7179962d7.png">
and below is the log screenshot.
<img width="1435" alt="Screenshot 2023-02-09 at 5 21 05 PM" src="https://user-images.githubusercontent.com/94165474/217805902-2909847e-5cb2-4b94-a545-85005954f39d.png">

When I try to login with a valid Active director credential I do get error that user password is not correct but there are no logs generated in debug screen.
<img width="1440" alt="Screenshot 2023-02-09 at 5 21 43 PM" src="https://user-images.githubusercontent.com/94165474/217806239-daa47c90-a2ee-4d0e-ba2d-6d88047fd9b2.png">

Please help me figure out what could be the reason. 

### Reproduction steps

1. I am trying to login with a valid AD credentials.
2.
3.
...

### Expected behavior

User should have been allowed to login on to the portal without any challenge as user is using correct credentials.

### Screenshots

Screenshots are updated in description of ticket. debug mode is on.
<img width="1440" alt="Screenshot 2023-02-09 at 5 15 20 PM" src="https://user-images.githubusercontent.com/94165474/217806922-36f9b831-0ba7-46d0-9543-ef1b8bc33da3.png">

### Snipe-IT Version

 Version v6.0.14 - build 9038 (master

### Operating System

Ubuntu

### Web Server

Apache

### PHP Version

7.4

### Operating System

macOS

### Browser

Chrome

**Comment by itakestime** [comment]
You don't have any validation errors with your LDAP filter string? I had some which prevented us from logging in (even though they were valid, SnipeIT recommended I fix them up)

What do you have set in the `Username Field` ? Whatever this is set to will need to match the relevant fields in your LDAP.

If you have this set to `sAMAccountName` then you'll need to log in with `user.name` instead of `user.name@domain.com`

**Comment by Pkumar22in** [comment]
@itakestime No, I don't have any errors in LDAP filter string, as this setup is running for last 2 years and was working perfectly fine. These issues started coming after recent upgrade. I have multiple instances running but I have this issue in couple of them. attached is the screenshot of settings I am running with. 
<img width="905" alt="Screenshot 2023-02-20 at 11 07 29 AM" src="https://user-images.githubusercontent.com/94165474/220017857-bf7267a7-5d37-48c5-a1ac-dbae04c25a03.png">

**Comment by snipe** [comment]
Hi there - sorry for the freakishly long time in responding here. This one fell off our radar :( 

Since the time this issue was opened, a ton of changes have been made in Snipe-IT. If you're still seeing this issue in the current version, please open a new issue that includes:

- Version of Snipe-IT you're running
- Specific errors you're seeing in the UI and/or in the browser console and error logs
- Steps to reproduce 
- Whether or not you can reproduce this on the demo (https://demo.snipeitapp.com)

Thanks - and again, sorry for the very late reply. We're doing some cleanup of old issues, so we're rediscovering some older things that didn't get addressed the first time around. 

![BeakerEmbarrased](https://github.com/user-attachments/assets/8533ac66-7970-4954-9cd3-2cc1147d4d41)

<blockquote><img src="https://demo.snipeitapp.com/favicon.ico" width="48" align="right"><div><strong><a href="https://demo.snipeitapp.com/">Snipe-IT Asset Management Demo</a></strong></div></blockquote>
